docs/releases.rst

```diff
@@ -0,0 +1,98 @@
+.. _releases:
+
+Releases
+=========
+
+Next planned releases
+---------------------
+
+* Sprint June 10 - June 30 2022 release: July 5th

```

Comment:
this will go stale immediately, i.e. it already has. I suggest we remove dates from here